### DISCUSSION TITLE
How to recover erasuring data if minio servive crash

### DISCUSSION THREAD

**Discussion by Smile-Autra** [minio/minio#14749] [Q&A]
How to recover erasuring data if minio servive crash

**Comment by ravindk89** [comment]
Distributed MinIO can auto-heal as long as you haven't lost quorum - see [Recover after Hardware Failure](https://docs.min.io/minio/baremetal/installation/restore-minio.html) for guides on replacing/repairing drives/nodes if there is a failure.  

If you lose too many disks/nodes where there is data loss - specifically more than the configured erasure code supports - if you have replication configured, you can use the remote MinIO deployment as a healthy source from which to [resynchronize data](https://docs.min.io/minio/baremetal/replication/server-side-replication-resynchronize-remote.html)
